lib/logflare/endpoints/resolver.ex

```diff
@@ -19,7 +19,8 @@ defmodule Logflare.Endpoints.Resolver do
   Returns the resolved pid.
   """
   def resolve(%Logflare.Endpoints.Query{id: id} = query, params) do
-    :syn.lookup(:endpoints, {id, params})
+    {:via, :syn, {:endpoints, {query.id, params}}}
+    |> GenServer.whereis()
```

Comment:
@chasers with this change I believe we now only have two possible responses, it should be either a pid or nil value :warning: 

cc/ @Ziinc